### DISCUSSION TITLE
2005/11/16/%E3%80%8Ano-excuse%E3%80%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#59] [Announcements]
# 2005/11/16/%E3%80%8Ano-excuse%E3%80%8B

二次世界大战以后，在世界500强企业里，有大约1000名董事长，2000名副董事长，5000多名总经理毕业自同一所学校。没有任何其他一所学校可以与之相比。这所学校就是西点军校。

https://blog.qizhen.xyz/2005/11/16/%E3%80%8Ano-excuse%E3%80%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Yume
2011年11月18日 17:43 编辑
嗯，我有读过这本书，这本书的前提条件在大部分的情况下会把国内人坑死。

回复
Billy
2009年06月27日 18:13 编辑
我上 amazon.com 找不到這本書,是偽書o

回复
Edward
2007年05月31日 08:37 编辑
你這本是偽書Ferrar Cape作者並不存在真正原作是Brace E. Barber請自行查證請順便了解一下「偽書」可能造成的危害

回复
alla
2006年10月11日 23:10 编辑
去年买了本送给一个妹妹看了，自己都没看下，周末去买本看看咯

回复
Unknown
2005年12月27日 20:54 编辑
我真佩服自己的眼力 啧啧！
